asset_dashboard/static/js/components/map_utils/EditControl.js

```diff
@@ -0,0 +1,148 @@
+// Code source: https://github.com/alex3165/react-leaflet-draw/blob/master/src/EditControl.js
```

Comment:
Can you use a permalink here instead of the master ref, which might change?

Reply:
done so here: efb9c07